faith_presentation/src/commonMain/kotlin/net/thechance/mena/faith/presentation/feature/quran/search/SearchInteractionListener.kt

```diff
@@ -0,0 +1,8 @@
+package net.thechance.mena.faith.presentation.feature.quran.search
+
+interface SearchInteractionListener {
+    fun onQueryChange(query: String)
+    fun clearQuery()
```

Comment:
What do you think about renaming `clearQuery()` to be `onClearQuery()` to follow consistency
```suggestion
    fun onClearQuery()
```